README.md

```diff
@@ -4,19 +4,28 @@
 [![Crates.io](https://img.shields.io/crates/v/autometrics.svg)](https://crates.io/crates/autometrics)
 [![Discord Shield](https://discordapp.com/api/guilds/950489382626951178/widget.png?style=shield)](https://discord.gg/kHtwcH8As9)
 
-Autometrics provides a macro that makes it easy to instrument any function with the most useful metrics: request rate, error rate, and latency. It then uses the instrumented function names to generate powerful Prometheus queries to help you identify and debug issues in production.
+Autometrics provides a macro that makes it trivial to instrument any function with the most useful metrics: request rate, error rate, and latency.
 
-## Features
+Autometrics then uses your function details to generate powerful Prometheus queries that help you identify and debug issues in production.
```

Comment:
Not sure if you need this here? You already say the same thing in the benefits section,

Reply:
Hmm, interesting.

Somehow, I think we need to get across the key power/benefit up front so people see that it's something they want (or at least something that seems intriguing) without reading all the features. Without this up here, I wonder if the intro sells it enough.

Reply:
Alright, I tried removing it in https://github.com/autometrics-dev/autometrics-rs/pull/112/commits/ca74855ba2949f5452842f1fd22efb9bf80e21a3